skimage/color/tests/test_colorconv.py

```diff
@@ -226,6 +226,15 @@ def test_rgb2grey(self):
 
         assert_equal(g.shape, (1, 1))
 
+    def test_rgb2grey_contiguous(self):
+        x = np.random.rand(10, 10, 3)
```

Comment:
Can you take the transpose here before sending it in so the input is not C-Contiguous?